os/x11/window.h

```diff
@@ -24,7 +24,7 @@
 #include <X11/Xatom.h>
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
-
+#include <set>
```

Comment:
This `#include` can go to the `window.cpp` file (because we aren't using `std::set` here in the .h)

Reply:
Done, I've just seen it after force pushing first commit :D